end_to_end/fraud_detection/3-mitigate-bias-train-model2-registry-e2e.ipynb

```diff
@@ -4,35 +4,34 @@
    "cell_type": "markdown",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #22.    # if 'clarify_expl_job_name' not in locals():
</pre><p>Style nit: commented lines don't get formatted, use double quotes</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3390/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Fixed</p>